lib/rubocop/cop/sorbet/signatures/enforce_signatures.rb

```diff
@@ -67,48 +65,111 @@ def scope(node)
 
         def check_node(node)
           scope = self.scope(node)
-          unless @last_sig_for_scope[scope] || has_rbs_comment?(node)
-            add_offense(
-              node,
-              message: "Each method is required to have a signature.",
-            ) do |corrector|
-              autocorrect(corrector, node)
+          sig_node = sig_checker.signature_node(scope)
+          rbs_node = rbs_checker.signature_node(node)
+
+          case signature_style
+          when "rbs"
+            # RBS style - only RBS signatures allowed
+            if sig_node
+              add_offense(
+                sig_node,
+                message: "Use RBS signature comments rather than sig blocks.",
+              )
```

Comment:
Style, I think this fits under the limit:

```suggestion
              add_offense(sig_node, message: "Use RBS signature comments rather than sig blocks.")
```

It will make the whole `case/when` easier to read.